app/src/main/java/org/p2p/wallet/history/ui/detailsbottomsheet/HistoryTransactionDetailsContract.kt

```diff
@@ -25,6 +25,6 @@ interface HistoryTransactionDetailsContract {
     }
 
     interface Presenter : MvpPresenter<View> {
-        fun load()
+        fun load(transactionId: String)
```

Comment:
isTransactionId string in Base58 or Base64?
better to use types instead of raw String

Reply:
I don't know, transaction for Solana is it's signature, it's string type, and moonpay transactionId string type